tests/bin/get_att.c

```diff
@@ -0,0 +1,16 @@
+#include "libc.h"
+#include <errno.h>
+
+int main(void) {
+    // TODO: Good buffer length?
+    unsigned char nonce[512];
+    unsigned char buf[512];
+    size_t technology;
+    size_t* tech_ptr = &technology;
+
+    ssize_t rax = get_att(nonce, sizeof(nonce), buf, sizeof(buf), tech_ptr);
+
+    // TODO: Currently this should return ENOSYS. However, this will change
+    // as get_att() is implemented for each technology.
+    return errno != ENOSYS;
```

Comment:
You also need to check the return value of `get_att()` (which should be `-1` for `errno` to be set).